react/wbooks/src/app/components/BookInfo/styles.module.scss

```diff
@@ -0,0 +1,31 @@
+@import '../../../scss/colors';
+
+.author {
+  color: $gray;
+  font-size: 14px;
+}
+
+.container {
+  background-color: $white;
+  box-shadow: 4px 4px 10px 0 $lighten-darken;
+  flex-grow: 1;
```

Comment:
this is making all the box grow when there is only one book in a row and its ignoring the width you declared